api/saml/configuration/model.py

```diff
@@ -64,6 +65,64 @@ class SAMLConfiguration(ConfigurationGrouping):
         format="narrow",
     )
 
+    patron_id_use_name_id = ConfigurationMetadata(
+        key="saml_patron_id_use_name_id",
+        label=_("Patron ID: Use SAML NameID"),
+        description=_(
+            "Boolean value indicating whether SAML NameID should be searched for a unique patron ID: "
+            "<br>"
+            "- <b>0</b> means that NameID won't be used, "
+            "<br>"
+            "- <b>1</b> (default) means that Circulation Manager will scan NameID in search for a patron ID."
```

Comment:
Suggest appending the following to value 1 for clarity: "If found, it will supersede any SAML Attributes selected in the next section."

Reply:
Added the suggested line.

Reply:
However, it might be confusing. NameID used only when there are _NO_ SAML attributes satisfying the conditions specified by the admin because it has the lowest priority.